run_meson_command_tests.py

```diff
@@ -143,7 +143,7 @@ def test_meson_installed(self):
         os.environ['PATH'] = str(bindir) + os.pathsep + os.environ['PATH']
         self._run(python_command + ['setup.py', 'install', '--prefix', str(prefix)])
         # Fix importlib-metadata by appending all dirs in pylibdir
-        PYTHONPATHS = [pylibdir] + [x for x in pylibdir.iterdir()]
+        PYTHONPATHS = [pylibdir] + list(pylibdir.iterdir())
```

Comment:
Stuff like this is interesting and fun. List incurs a name lookup and function call, but exchanges it for being able to work with the raw iterator. A list comprehension is built-in but has to run bytecode for each step.

I haven't profiled it or anything. But this seems like the kind of thing I'd half expect the Faster CPython project project to target as an optimization opportunity.

Reply:
This change was reverted.

Reply:
You don't need to revert it (there are a bunch more of them anyway :P). This was just me commenting as an aside.